Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -37,7 +37,12 @@ configuration:
   type: 13
 - defaultvalue: 1 days
   display: First fetch timestamp, in days.
-  name: fetch_time
+  name: first_fetch
```

Comment:
Breaking b.c.

Reply:
As new convention from demisto-sdk. If we did not change this it will give an error to rename it to first_fetch.